app/src/main/java/com/gh4a/service/RepositoryService.java

```diff
@@ -0,0 +1,80 @@
+package com.gh4a.service;
+
+import android.content.Context;
+import com.gh4a.DefaultClient;
+import com.gh4a.Gh4Application;
+import com.gh4a.utils.HtmlUtils;
+import com.gh4a.utils.RxTools;
+
+import org.eclipse.egit.github.core.Repository;
+import org.eclipse.egit.github.core.RepositoryId;
+import org.eclipse.egit.github.core.client.GitHubClient;
+import org.eclipse.egit.github.core.client.RequestException;
+import org.eclipse.egit.github.core.service.ContentsService;
+import org.eclipse.egit.github.core.service.IssueService;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Locale;
+
+import io.reactivex.Observable;
+
+public class RepositoryService {
+    public static Observable<String> loadReadme = null;
```

Comment:
I would prefer if this was set to private. Instead add an additional method for resetting it to null. That way we'll avoid null pointer exceptions when someone tries to use this subscription directly instead of method.

Reply:
just gotta check how to undo `cache` operator on the observable, or I wont be able fetching data from the network again :S :P